sceptre/cli/helpers.py

```diff
@@ -173,6 +189,23 @@ def _generate_text(stream):
     return stream
 
 
+def process_template(
+    template: dict, output_format: str, type: str = "template"
+) -> None:
+    """
+    Helper to write templates and configs used in dump commands.
+
+    :param template: the template.
+    :param output_format: The format to print the output as. Allowed values: \
+    :param type: either template or config used only in the file name.
+    """
+    stack_name = list(template.keys())[0]
+    file_path = Path(".dump") / stack_name / f"{type}.{output_format}"
+    logger.info(f"{stack_name} dumping {type} to {file_path}")
+    write(template[stack_name], output_format, no_colour=True, file_path=file_path)
+    logger.info(f"{stack_name} done.")
```

Comment:
I don't think we need both of these log messages, at least not both at the `info` level. 

I think having the initial message would be very helpful in the case that writing fails for some reason (not sure why that would be the case, though...). So I think the first log message should be emitted at the `debug` level rather than `info` and the second log message should include the file_path as well and be something like: `f'{stack_name} dumped to file at {file_path}'`.

Reply:
@jfalkenstein the message is proposed by @zaro0508. I would add that without a second logger message after dump has completed, the first will be probably not noticed.

Reply:
have a look now.